servicetalk-grpc-protobuf/src/main/java/io/servicetalk/grpc/protobuf/ProtoBufSerializationProviderBuilder.java

```diff
@@ -172,7 +172,7 @@ public GrpcSerializationProvider build() {
             @SuppressWarnings("unchecked")
             HttpSerializer<T> httpSerializer = serializersForType.get(codec);
             if (httpSerializer == null) {
-                throw new MessageEncodingException("Unknown encoding: " + codec.name());
+                throw new MessageEncodingException(codec.name().toString());
```

Comment:
Full message is generated inside a ctor

Reply:
Decided to open a separate PR #1940 because we should also cherry-pick this into 0.41